3.6/programs-web-interface-cluster.md

```diff
@@ -7,6 +7,8 @@ Cluster
 
 The cluster section displays statistics about the general cluster performance.
 
+ArangoDB is also available as [cloud service - **ArangoDB Oasis**](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic){:target="_blank"}. Fire up your cluster in just a few clicks with the [14-day free trial](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic){:target="_blank"}.
```

Comment:
The entire chapter is hopelessly outdated, better not add something there.

Reply:
OK. Let's remove it.